portal/guides/on-demand-minting.mdx

```diff
@@ -1,13 +1,15 @@
 ---
-title: "Using signature based minting"
-author: "Raza Zaidi"
+title: "Minting NFTs on demand using signatures"
+author: "Raza Zaidi and Ayush Pathak"
 authorAvatar: "/assets/portal/authors/raza-zaidi.jpg"
+authorAvatar2: "/assets/portal/authors/ayush.jpg"
 authorTwitter: "https://twitter.com/razacodes"
-draft: true
+authorTwitter2: "https://twitter.com/ayshptk"
+draft: false
 tags: [nft-drop, signature, mint]
```

Comment:
tag should be `nft-collection` instead of drop

Reply:
done